frontend/src/components/GForceDeleteCluster.vue

```diff
@@ -0,0 +1,120 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<template>
+  <v-list>
+    <v-list-item-subtitle>
+      Created By
+    </v-list-item-subtitle>
+    <v-list-item-title>
+      <g-account-avatar
+        :account-name="shootCreatedBy"
+        :size="22"
+      />
+    </v-list-item-title>
+  </v-list>
+  <p>
+    Type <span class="font-weight-bold">{{ shootName }}</span> below to confirm the forceful deletion of the cluster.
+  </p>
+  <g-expand-transition-group>
+    <v-alert
+      v-if="!confirmed"
```

Comment:
the alert should not disappear, as the checkbox is referring to the text above which then disappears